getting-started/cloudflare-pages.md

```diff
@@ -121,7 +121,7 @@ bun run dev
 
 ## 4. Deploy
 
-If you have a Cloudflare account, you can deploy to Cloudflare.
+If you have a Cloudflare account, you can deploy to Cloudflare. In `package.json` on line 7, replace `$npm_execpath` with your package manager of choice. Then run:
```

Comment:
"line 7" may change, so delete it and just write "`$npm_execpath` to be changed".